reactive/docker.py

```diff
@@ -264,6 +265,11 @@ def install_from_nvidia_apt():
                 "9DC858229FC7DD38854AE2D88D81803C0EBFCD88"]:
         add_apt_key(key)
 
+    # Install key for nvidia-docker. This key changes frequently
+    # ([expires: 2019-09-20]) so we should do what the official docs say and
+    # not try to get it through its fingerprint.
+    add_apt_key_url("https://nvidia.github.io/nvidia-container-runtime/gpgkey")
```

Comment:
do we still need the 2 keys above that use `add_apt_key` now that we do this url thing?

Reply:
You are right, we can remove the expired one:
```
> gpg --recv-keys C95B321B61E88C1809C4F759DDCAE044F796ECB0
gpg: key DDCAE044F796ECB0: "NVIDIA CORPORATION (Open Source Projects) <cudatools@nvidia.com>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
> gpg -a --export "NVIDIA CORPORATION (Open Source Projects)" | gpg --list-packets
# off=0 ctb=99 tag=6 hlen=3 plen=525
:public key packet:
        version 4, algo 1, created 1506629681, expires 0
        pkey[0]: [4096 bits]
        pkey[1]: [17 bits]
        keyid: DDCAE044F796ECB0
# off=528 ctb=b4 tag=13 hlen=2 plen=64
:user ID packet: "NVIDIA CORPORATION (Open Source Projects) <cudatools@nvidia.com>"
# off=594 ctb=89 tag=2 hlen=3 plen=568
:signature packet: algo 1, keyid DDCAE044F796ECB0
        version 4, created 1506629681, md5len 0, sigclass 0x13
        digest algo 10, begin of digest 08 7a
        hashed subpkt 2 len 4 (sig created 2017-09-28)
        hashed subpkt 27 len 1 (key flags: 0F)
        hashed subpkt 11 len 5 (pref-sym-algos: 9 10 8 7 3)
        hashed subpkt 21 len 4 (pref-hash-algos: 10 9 8 11)
        hashed subpkt 22 len 4 (pref-zip-algos: 2 3 1 0)
        hashed subpkt 30 len 1 (features: 01)
        hashed subpkt 23 len 1 (keyserver preferences: 80)
        subpkt 16 len 8 (issuer key ID DDCAE044F796ECB0)
        data: [4096 bits]
# off=1165 ctb=b9 tag=14 hlen=3 plen=269
:public sub key packet:
        version 4, algo 1, created 1506630745, expires 0
        pkey[0]: [2048 bits]
        pkey[1]: [17 bits]
        keyid: C45B1676A04EA552
# off=1437 ctb=89 tag=2 hlen=3 plen=836
:signature packet: algo 1, keyid DDCAE044F796ECB0
        version 4, created 1506630745, md5len 0, sigclass 0x18
        digest algo 10, begin of digest f4 3a
        hashed subpkt 2 len 4 (sig created 2017-09-28)
        hashed subpkt 27 len 1 (key flags: 02)
        hashed subpkt 9 len 4 (key expires after 1y0d0h0m)
        subpkt 16 len 8 (issuer key ID DDCAE044F796ECB0)
        subpkt 32 len 284 (signature: v4, class 0x19, algo 1, digest algo 10)
        data: [4096 bits]
```